src/classes/STG_PanelRDHealthCheck.cls

```diff
@@ -126,10 +141,9 @@ public inherited sharing class STG_PanelRDHealthCheck {
 
         result.withRecurringDonationScope();
 
-        validateSupportedPicklistValues(supported.installmentPeriods, 'npe03__Installment_Period__c',
-            System.Label.healthDetailsInvalidPicklistValue, System.Label.healthSolutionUnsupportedPicklistValue);
+        validateStatusMappingSetting();
 
-        validateSupportedPicklistValues(supported.status, String.valueOf(npe03__Recurring_Donation__c.Status__c),
+        validateSupportedPicklistValues(supported.installmentPeriods, 'npe03__Installment_Period__c',
```

Comment:
This check on line 149 can be removed since it is duplicate of the `supported.installmentPeriods` check on line 144 (the indentation is off on line 145). However, the validation of supported Status pickllist values should be added back with modification to validate required Active, Lapsed and Closed picklist values.